.expeditor/verify.pipeline.yml

```diff
@@ -11,10 +11,32 @@ expeditor:
 
 # why do we test on Linux? It's the easiest way to run specs quickly on multiple ruby versions so we test on Windows and then on various ruby releases using Linux agents
 steps:
-- label: lint-chefstyle
+- label: run-specs-ruby-2.7-windows
```

Comment:
We should keep the `lint-chefstyle` job running on a linux system. That will run `rake style` which just makes sure ruby syntax formatting is correct. Try running it with `image: ruby:3.1`.